src/store/interface.tsx

```diff
@@ -0,0 +1,32 @@
+export interface IChat {
+    id : number;
+    userid : number;

```

Comment:
```suggestion
    chatId : number;
    userId : number;
```

변수명, 타입명들은 userId, userName, roomId 과 같이 카멜케이스로 작성하는 것을 권장해요!!
추가로, userid와 구분되는 네이밍이 있으면 좋을 것 같아요!! id -> chatId와 같이..!